packages/application-generic/src/services/in-memory-provider/providers/elasticache-cluster-provider.ts

```diff
@@ -48,7 +48,9 @@ export const getElasticacheClusterProviderConfig =
       keepAlive: process.env.REDIS_CLUSTER_KEEP_ALIVE,
       family: process.env.REDIS_CLUSTER_FAMILY,
       keyPrefix: process.env.REDIS_CLUSTER_KEY_PREFIX,
-      tls: process.env.REDIS_CLUSTER_TLS as ConnectionOptions,
```

Comment:
Are we deprecating the `REDIS_CLUSTER_TLS` env var, or was this a mistake earlier?

Reply:
@rifont what do you mean? We needed to do this so that we can connect to azure redis with ssl.

Reply:
@rifont that's a bit of a legacy env var we left to reuse. It comes from before when there was just an instance of Redis for all the platform.